### ISSUE TITLE
Suggestion: Multiple Encryption Keys

### ISSUE THREAD

**Issue by RuinTD** [Aizistral-Studios/No-Chat-Reports#245]
### Idea

There should be an option to store multiple encryption keys, allowing the user to switch between them for encryption, and the application would attempt each key for decryption.

### Reasoning

This would make the encryption feature a lot more useful, especially when using custom keys.

### Other Information

* The default key should be hard-coded into the list.
* You should be able to rename keys, so they can be later identified,
* There could be options to assign keys for specific servers, or maybe even for specific users being messaged.
* Caesar might be hard (impossible?) to allow multiple keys for decryption.
  * Probably only allow one Caesar key. It isn't secure anyway.

**Comment by RedstoneParkour** [comment]
Instead of trying each and every key (which can take time as the amount of keys increase), it might be possible to store a very short hash in the message itself to identify the key.

**Comment by Madis0** [comment]
> There could be options to assign keys for specific servers, or maybe even for specific users being messaged.

I agree with this part, I think the list wouldn't be too useful when it is just arbitrary.
Similarly encryption could be global on one server and PM-only on another.

**Comment by IlyaTheSawmill** [comment]
You can set a key to send message, and you can select specific key by sending with holding some keypress(such as shift).
Select what keys to decrypt.

**Comment by Madis0** [comment]
Encryption features were removed in v2.12.0, so this will not be considered. [See the alternatives at the wiki.](https://github.com/Aizistral-Studios/No-Chat-Reports/wiki/To-Encrypt-or-Not-To-Encrypt#alternatives)
